gamebench_api_client/models/dataframes/generic/abstract_generic.py

```diff
@@ -22,10 +21,12 @@ def __init__(self, **request_parameters):
             :param request_parameters: Dictionary passed in from the model Creator.  Contains information to
                 build and send a response.
         """
-        username_and_password = get_username_and_password()
+
         super().__init__()
-        self.authenticator = Authenticator(username_and_password)
-        self.mediator = GenericMediator(**request_parameters)
+        self.authenticator = Authenticator()
+        self.request_parameters = request_parameters
```

Comment:
This can be deleted as it's duplicated from the base class.

Reply:
Removed in dce6d57!